docs/resources/Guild.md

```diff
@@ -389,6 +389,9 @@ Adds a user to the guild, provided you have a valid oauth2 access token for the
 > info
 > All parameters to this endpoint except for `access_token` are optional.
 
+> info
+> The Authorization header must be Bot authorization, with the token belonging to the bot associated with the application.
```

Comment:
` with the token belonging to the bot associated with the application.` this may cause further confusion as it maybe mistaken as the `access_token`, not the Authorization. I believe just stating "This endpoint must be authorized with a bot within the guild" or something similar would suffice. 

Does the bot require INVITE permission? if it does maybe mention that too.

Reply:
>  "This endpoint must be authorized with a bot within the guild"

I've seen someone use a bot *not* associated with the application before and get confused by that also. I'd like to make it non ambiguous. Any suggestions?

> Does the bot require INVITE permission? if it does maybe mention that too.

Yes that's mentioned right above it " Requires the bot to have the CREATE_INSTANT_INVITE permission."

Reply:
Maybe "with _the application's_ bot" instead of just _a_ bot?